index.html

```diff
@@ -182,76 +237,151 @@ <h1 class="work-sample"> Profesional Art Printing Data </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+		</div>
+	</div>
+	</div>
 </section>
 <br> <br> <br>
 <section class="head">
 	<!-- Heading Section  -->
 	<div class="flex-about">
-	  <div class="about-me">
+	<div class="row">
+	  <div class="about-me col-7">
 	  	<h1 class="head-title"> 
 	    	About Me 
 	    </h1>
 	  	<p class="head-body">
 	  		Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
 	  	</p>
-	  	<div class="see-3">
-	  		
-	  	</div>
 	  	<a class="see-3" href="#">Get My Resume</a>
-	  	<div>
-	  		<br><br><br><br><br>
-	  		<hr>
-	  	</div>
 	  </div> 
-	  <div>
-	  	<br>
-	  	<br>
-	  	<ul class="lang">
-	  		<li class="lang-title"> <img src="assets/images/rectangle.png" alt="rec">Language</li>
-		  	<li>JavaScript</li>
-		  	<li>Ruby</li>
-		  	<li>HTML</li>
-		  	<li>CSS</li>
-	  	</ul>
-	  	<ul class="lang">
-	  		<li class="lang-title"> <img src="assets/images/rectangle.png" alt="rec">Frameworks</li>
-		  	<li>Ruby On Rail</li>
-		  	<li>RSPec</li>
-		  	<li>Copy Bara</li>
-		  	<li>Selenium</li>
-	  	</ul>
-	  	<ul class="lang">
-	  		<li class="lang-title"> <img src="assets/images/rectangle.png" alt="rec">Skills</li>
-		  	<li>CodeKit</li>
-		  	<li>GitHub</li>
-		  	<li>Code Pen</li>
-		  	<li>GitLab</li>
-		  	<li>Terminal</li>
-	    </ul>
+	  <div class="col-5 hide">
+		<img width="100%" src="assets/images/about.png" alt="rec">
+	  </div> 
+	</div>
+		<div class="row">
+			<br> <br> <hr> <br> <br>
+		<div class="col-4">
+			<ul class="lang">
+				<li class="lang-title"> <img src="assets/images/rectangle.png" alt="rec">Language</li>
+				<li>JavaScript</li>
+				<li>Ruby</li>
+				<li>HTML</li>
+				<li>CSS</li>
+			</ul>
+		</div>
+		<div class="col-4">
+			<ul class="lang">
+				<li class="lang-title"> <img src="assets/images/rectangle.png" alt="rec">Frameworks</li>
+				<li>Ruby On Rail</li>
+				<li>RSPec</li>
+				<li>Copy Bara</li>
+				<li>Selenium</li>
+			</ul>
+		</div>
+		<div class="col-4">
+			<ul class="lang">
+				<li class="lang-title"> <img src="assets/images/rectangle.png" alt="rec">Skills</li>
+				<li>CodeKit</li>
+				<li>GitHub</li>
+				<li>Code Pen</li>
+				<li>GitLab</li>
+				<li>Terminal</li>
+			</ul>
+		</div>
+
 	  </div>  	
 	</div>
 </section>
 <section class="footer">
 	<div class="flex-footer">
-		<div>
+	<div class="row">
+		<div class="col-5">
 			<p class="footer-datail">
 				I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
 		 	</p>
-		</div>
-		<div>
+    	</div>
+		<div class="col-7">
```

Comment:
- [ ] Form section design(mobile view) different than original design please fix it.

![image](https://user-images.githubusercontent.com/38635158/173109101-648b50a6-9d59-4e68-a601-fc187e756a23.png)
___
Original
![image](https://user-images.githubusercontent.com/38635158/173110070-577b7f20-b0dc-4e75-9635-944668eef87f.png)

Reply:
Thank you for the corrections, I applied that. Please check it by Inspect -> mobile devices -> Responsive